Added DEI.md file for CHAOSS project badging, to highlight Django's dedication to diversity, equity and inclusion. (#2)
django/.github: DEI.md

# Diversity, Equity, and Inclusion Project Statement

<!---
The DEI.md file was originally created in the CHAOSS project. This comment provides attribution of that work as defined under the MIT license. 
-->

<!---
Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating your DEI.md file
-->

> <img src="django-logo.png" alt="Django" style="width:200px;"/>

Reviewer: Can we use SVG to look good at all resolutions and zoom levels?

Author: I'll see if I can find one 👍

Author: Updated ⭐

* [Statistics of Code of Conduct reports](https://github.com/django/code-of-conduct/blob/main/statistics.md) are publicly available.
* DjangoCon talks are publicly available - [DjangoCon Europe](https://www.youtube.com/@djangoconeurope), [DjangoCon US](https://www.youtube.com/@DjangoConUS)
           
### [Newcomer Experiences](https://chaoss.community/?p=4891)
The newcomer experience is addressed in our project through a variety of different efforts. Through these efforts, we aim to support the newcomer experience for all new members. Specific efforts in our project include:
* A quick start guide for newcomers is available on our website and made available through the main website navigation.
* Django has an automatic welcome message for first time contributors to Django who are awaiting a response to a merge request.
* Django's contributor documentation includes a full walkthrough tutorial for newcomers to make their first contribution. This includes setting up the development environment, making code changes, documentation updates and ways to find support.

> * Django conferences welcome and prioritise new speakers.

Reviewer 2: DjangoCon's offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained. (my wording isn't great here so far)

* Django Software Board members have limited terms which require a renewal via a community vote. Everyone in the community can run to be a member of the Board.
* Django is community run. All decision-making/leadership responsibilities are help by groups rather than individuals.
* The Code of Conduct committee enforces the code of conduct consistently.

Our project recognizes that the inclusion of the DEI.md file and the provided reflection on the specific DEI metrics does not ensure community safety nor community inclusiveness. The inclusion of the DEI.md file signals that we, as a project, are committed to centering DEI in our project and regularly reviewing and reflecting on our project DEI practices.

If you do not feel that the DEI.md file appropriately addresses concerns you have about community safety and inclusiveness, please let us know. You can do this by reporting your concerns to our Code of Conduct team (provide a link to your code of conduct team) to raise concerns you have. (If there is no Code of Conduct team, please replace the last sentence with an appropriate reporting process). 

> Last Reviewed: [Enter Date]

Reviewer 3: Is that a no new line at end of file?

-->

<img src="https://static.djangoproject.com/img/logos/django-logo-negative.svg" alt="Django" style="width:200px;"/>

Scope of the DEI.md File

Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) DEI metrics. 

### [Project Access](https://chaoss.community/?p=4953)
Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for all. Specific efforts in our project include:
* Our Django Fellows span global time zones and work asynchronously with contributors across the world.
* The Django documentation and admin has multiple translations.
* All areas of Django are considered open to community contributions. 
* Django Software Foundation membership, which recognises contributors to Django, is open to non-code contributors, including contributions through events, blog posts, and comments in issues and merge requests.
* We have an Accessibility Team dedicated to improving the accessibility of Django.

> * DjangoCon's offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained.

Reviewer 4: @sarahboyce I don't know if this is worth mentioning somewhere but Django sponsors can make extra donation dedicated to this purpose in DjangoCons. 
For example, it was the case for DjangoCon Europe last year: https://2023.djangocon.eu/sponsors/information/ or even custom / Opportunity Grant Sponsor in DjangoCon US: https://2023.djangocon.us/sponsors/information/

Author: I've added a note to draw attention to this - thank you Sarah ❤️

> * DjangoCon's offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained.

Reviewer 4: @sarahboyce I don't know if this is worth mentioning somewhere but Django sponsors can make extra donation dedicated to this purpose in DjangoCons. 
For example, it was the case for DjangoCon Europe last year: https://2023.djangocon.eu/sponsors/information/ or even custom / Opportunity Grant Sponsor in DjangoCon US: https://2023.djangocon.us/sponsors/information/

> * DjangoCon's of recent years offer a free crèche and private facilities for breastfeeding.

Author: @tim-schilling I have added these lines here within the Project Access section. Is there anything else around DjangoCon's you want to draw attention to?

Reviewer 2: Sorry, I thought I responded. "childcare" may be more accessible to folks than "crèche". And I don't think it was free, it was a reimbursement up to some value. So maybe "DjangoCon's make it a priority to offer childcare reimbursements and private facilities for breastfeeding."

Reviewer: Also drop the apostrophes: “DjangoCons”

* A quick start guide for newcomers is available on our website and made available through the main website navigation.
* Django has an automatic welcome message for first time contributors to Django who are awaiting a response to a merge request.
* Django's contributor documentation includes a full walkthrough tutorial for newcomers to make their first contribution. This includes setting up the development environment, making code changes, documentation updates and ways to find support.
* Django conferences welcome and prioritise new speakers.
  
### [Inclusive Leadership](https://chaoss.community/?p=3522)
Inclusive leadership is addressed in our project through a variety of different efforts. Through these efforts, we aim to support leadership opportunities for project members with an interest. Specific efforts in our project include: 
* Django Software Foundation (DSF) Board members have limited terms which require a renewal via a community vote. Everyone in the community can run to be a member of the Board.
* Django is community run. All decision-making/leadership responsibilities are help by groups rather than individuals.

> * The [DSF Board](https://www.djangoproject.com/foundation/#board) has included a diverse global range of people over time, continues to do so. At the time of writing, 3 out of 7 board members are female.

Author: @carltongibson I've added a note here with a link to the current board. I don't think the current board is majority female but I don't know everyone personally - can you check if 3 out of 7 is correct?

Reviewer 5: 3 of 7 is correct as a minimum through December 2024. Possibly higher after the current election.

Given the will change year to ear, and the "Last Reviewed" date at the bottom of the page is there a plan as to who will own the updating of this page?

Reviewer 5: My mistake. I received feedback that this does not accurately represent gender identity of the board. Instead of stating a diversity breakdown for the current board and attempting to keep it up to date could we have something like `The DSF membership has historically and continues to elect  gender, racially, geographicly diverse DSF Boards(https://www.djangoproject.com/foundation/#board). ` I'm sure the diversity language can be improved or better enumerated

Reviewer 6: I strongly agree to not to call out any concrete minority group. My proposal:

```suggestion
* The DSF membership has historically and continues to elect members for the [DSF Board](https://www.djangoproject.com/foundation/#board) that are diverse in terms of gender, age, race, geography, and more.
```

Author: I've updated the phrasing to be more generic.
At the bottom of the document there is a way to contact to notify if the document is out of date (this contact is currently the coc committee as suggested by CHAOSS but we can update).
I have ideas around creating a working group to cover DEI activities in general and they would review this doc periodically, however, most of this is written in a way that it shouldn't be out of date too quickly (so we can manage without a working group for now).

Author: Ah sorry, I missed the extra messages. Updated 👍

# Diversity, Equity, and Inclusion Project Statement

<!---
The DEI.md file was originally created in the CHAOSS project. This comment provides attribution of that work as defined under the MIT license. 
-->

<!---
Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating your DEI.md file
-->

<img src="https://static.djangoproject.com/img/logos/django-logo-negative.svg" alt="Django" style="width:200px;"/>

Scope of the DEI.md File

> Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) DEI metrics. 

Reviewer 6: Shall we wrap at 79cols?

### [Project Access](https://chaoss.community/?p=4953)

> Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for all. Specific efforts in our project include:

Reviewer 6: ```suggestion
Project access is addressed through various efforts, with the aim of supporting access for all. Specific efforts in our project include:
```

Author: This text is generic based off the CHAOSS template. Although I think we can update this, I want to keep it the same to align with other projects.

* Our Django Fellows span global time zones and work asynchronously with contributors across the world.

> * The Django documentation and admin has multiple translations.

Reviewer 6: I'm proposing the following change to ensure we include translations that are done for other parts of the framework (besides admin): such as validation strings, error messages, management commands help messages, template tags and filters, User management, etc.
```suggestion
* The Django documentation and the Django framework itself has multiple translations.
```

> * All areas of Django are considered open to community contributions. 

Reviewer 6: I would suggest adding one or two concrete items to exemplify what "areas" mean (I'm not sure myself :-))

Perhaps:
```suggestion
* All areas of Django are considered open to community contributions, including new features decision making, bug solutions brainstorming, etc. 
```

Author: I added some examples of what I mean, good idea 👍

> * Django Software Foundation membership, which recognises contributors to Django, is open to non-code contributors, including contributions through events, blog posts, and comments in issues and merge requests.

Reviewer 6: ```suggestion
* Django Software Foundation (also referenced as the DSF) membership, which recognizes contributors to Django, is open to non-code contributors, including contributions through events, blog posts, and comments in issues and merge requests.
```

* We have an Accessibility Team dedicated to improving the accessibility of Django.
* DjangoCons offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained.
* DjangoCons make it a priority to offer childcare reimbursements and private facilities for breastfeeding.

> * DjangoCon talks have human captioners which make talks more accessible - especially to those hard of hearing and language learners.

Reviewer 6: ```suggestion
* DjangoCon talks have in place human captioners which make talks more accessible - especially to those hard of hearing and language learners.
```

_Note: the extra provisions and opportunity grants are only possible at DjangoCons due to our generous sponsors. 
DjangoCons have diversity and accessibility packages which allow sponsors to specifically fund these efforts._
  
### [Communication Transparency](https://chaoss.community/?p=4957)

> Communication Transparency is addressed in our project through a variety of different efforts. Through these efforts, we aim to support transparency for all. Specific efforts in our project include: 

Reviewer 6: ```suggestion
Communication Transparency is addressed in our project through a variety of different efforts,with the aim of supporting transparency for all. Specific efforts in our project include: 
```

Author: (generic from template)

> * All project documentation is publicly available and [djangoproject.com](https://djangoproject.com).

Reviewer 6: ```suggestion
* All project documentation is publicly available in [djangoproject.com](https://djangoproject.com).
```

* Decisions that impact Django are shared via the Django Software Board [meeting minutes](https://www.djangoproject.com/foundation/minutes/).
* [Statistics of Code of Conduct reports](https://github.com/django/code-of-conduct/blob/main/statistics.md) are publicly available.

> * DjangoCon talks are publicly available - [DjangoCon Europe](https://www.youtube.com/@djangoconeurope), [DjangoCon US](https://www.youtube.com/@DjangoConUS)

Reviewer 6: I think we should include the Django ticket system, Django forums and discourse in this section as well -- they are public and while they require login, anyone can create tickets/triage a bug/see and post messages/comments, reply, etc

           
### [Newcomer Experiences](https://chaoss.community/?p=4891)

> The newcomer experience is addressed in our project through a variety of different efforts. Through these efforts, we aim to support the newcomer experience for all new members. Specific efforts in our project include:

Reviewer 6: ```suggestion
The newcomer experience is addressed in our project through a variety of different efforts, with the aim of supporting the newcomer experience for all new members. Specific efforts in our project include:
```

Author: This is also generic based off the CHAOSS template 👍

* A quick start guide for newcomers is available on our website and made available through the main website navigation.

> * Django has an automatic welcome message for first time contributors to Django who are awaiting a response to a merge request.

Reviewer 6: Shall we also include the specific channels in Discourse and the forum for seeking help?
Also, Djangonauts space!!!

* Django's contributor documentation includes a full walkthrough tutorial for newcomers to make their first contribution. This includes setting up the development environment, making code changes, documentation updates and ways to find support.
* Django conferences welcome and prioritise new speakers.
  
### [Inclusive Leadership](https://chaoss.community/?p=3522)

> Inclusive leadership is addressed in our project through a variety of different efforts. Through these efforts, we aim to support leadership opportunities for project members with an interest. Specific efforts in our project include: 

Reviewer 6: ```suggestion
Inclusive leadership is addressed in our project through a variety of different efforts, with the aim to supporting leadership opportunities for project members with an interest. Specific efforts in our project include: 
```

Author: (from template)

> * Django Software Foundation (DSF) Board members have limited terms which require a renewal via a community vote. Everyone in the community can run to be a member of the Board.

Reviewer 6: I would suggest we introduce the `DSF` acronym early and not repeat here.

* Django is community run. All decision-making/leadership responsibilities are help by groups rather than individuals.

> * The [DSF Board](https://www.djangoproject.com/foundation/#board) includes a diverse global range of folks. Notably, since 2019, 2-4 out of our 7 board members have been female.

Reviewer 6: I would personally not call out any concrete minority group. Using what Chaim suggested:
```suggestion
* The DSF membership has historically and continues to elect [boards](https://www.djangoproject.com/foundation/#board) that are diverse in terms of gender, age, race, geography, and more. 
```

under the MIT license. 
-->

<!---
Please use the DEI.md Guide at https://github.com/AllInOpenSource/ProjectBadging/blob/main/Guide.DEI.md when creating 
your DEI.md file
-->

<img src="https://static.djangoproject.com/img/logos/django-logo-negative.svg" alt="Django" style="width:200px;"/>

Scope of the DEI.md File

Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This 
reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) 
DEI metrics. 

### [Project Access](https://chaoss.community/?p=4953)
Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for 
all. Specific efforts in our project include:
* Our Django Fellows span global time zones and work asynchronously with contributors across the world.
* The Django documentation and the Django framework itself has multiple translations.
* All areas of Django are considered open to community contributions, including joining a translation team, conference 
organising or volunteering, running for the DSF board, Django ticket triage etc.

> * DSF membership, which recognizes contributors to Django, is open to non-code contributors, including contributions 

Reviewer 6: I have a worry that we are naturally referring to the `DSF` but the reader (at this point) may have no idea what that means or is. Could we please clarify the first time we use the acronym?

Author: Ah sorry I didn't mean to remove them all, updated the one above 👍
